AdyenEncryption/Extensions/StringExtension.swift

```diff
@@ -27,5 +27,17 @@ extension String {
 
         return data
     }
+    
+    internal func base64String() -> String {
```

Comment:
i think it would be good to somehow make sure this can be only called on base64URL strings. one idea is to change the name to `base64URLtoBase64`.  we can also add `base64toBase64URL` as an additional helper

Reply:
I made the function throws if its not a valid base64 string